src/extra/calibration.py

```diff
@@ -278,6 +280,17 @@ def dataset_obj(self, caldb_root=None) -> h5py.Dataset:
     def ndarray(self, caldb_root=None):
         return self.dataset_obj(caldb_root)[:]
 
+    def calcat_metadata(self, client=None):
```

Comment:
Name bikeshedding inc.

Mind just using `metadata()`? I could imagine this also being part of some better constant format, or having more metadata there over time.

Reply:
My thinking with putting 'calcat' in there was to give a hint that a) it might not be available if the constants you're looking at aren't in calcat, and b) it might require a request to calcat to get the information.

But it's a good point that some or all of this metadata may be available outside calcat. Even if we load `calibration_metadata.yml` from a correction, we have the begin validity timestamp, and it would be nice to keep that available even without the rest of the metadata. :thinking: